typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -254,6 +332,21 @@ export const getRenzoEZETHWarpConfig = async (): Promise<
                   },
                 ],
               },
+              hook: {
+                type: HookType.AGGREGATION,
```

Comment:
thinking of a couple things:

1. you called out that the infra checker doesn't support hook configs -- this will probably cause our warp checker job to fail due to this assert https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/sdk/src/router/HyperlaneRouterChecker.ts#L71. You can test this by running it for eg pzETH:
```
yarn tsx ./scripts/check/check-deploy.ts -e mainnet3  -m warp --warpRouteId PZETH/ethereum-swell-zircuit
```

So before this gets merged, we'll wanna make sure that this won't break the checker

2. Once we have hook checking, if the `tokenPrices.json` changes (we usually update when we do a new deploy batch), then iiuc we'll now have a checker violation because `getProtocolFee` will return something different.

Should we maybe create a copy of tokenPrices that are used in this config to avoid that situation? Otherwise we'll need to tolerate checker violations or update the protocol fee more frequently than we probably want

Reply:
for 1, I understand you haven't touched this code much, so lmk if you'd like some help from the backend team there